src/workers/InlineWorker.js

```diff
@@ -24,15 +32,15 @@ module.exports = function (content, url) {
         blob = new Blob([content]);
       }
 
-      return new Worker(URL.createObjectURL(blob));
+      return new createWorker(URL.createObjectURL(blob), options.isSharedWorker);
```

Comment:
Mistype?

Reply:
Confusing to see both `new` and `CreateWorker()`, that not looks like a constructor